graphics/timg/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup cmake 1.1
+PortGroup github 1.0
+
+revision 1
+
+categories graphics
+maintainers @urkle
```

Comment:
please also add an obfuscated e-mail address and, if there is no obvious reason not do so, add also "openmaintainer".